pkg/kp/rcstore/consul_store.go

```diff
@@ -341,7 +341,7 @@ func (s *consulStore) Watch(rc *fields.RC, quit <-chan struct{}) (<-chan struct{
 				case <-quit:
 				}
 			} else {
-				*rc = newRC
+				rc = &newRC
```

Comment:
as I understand it, [this will result in no externally-visible change to `rc`](http://play.golang.org/p/BOoyrK0lTa) - is that OK with you? If it is OK, how do callers of this function get an updated RC (or do they not care about having one?)?